inference/core/roboflow_api.py

```diff
@@ -183,16 +184,33 @@ def get_roboflow_model_data(
     endpoint_type: ModelEndpointType,
     device_id: str,
 ) -> dict:
-    api_url = _add_params_to_url(
-        url=f"{API_BASE_URL}/{endpoint_type.value}/{model_id}",
-        params=[
-            ("api_key", api_key),
+    api_data_cache_key = f"roboflow_api_data:{endpoint_type.value}:{model_id}"
```

Comment:
I am not sure if this is a good idea

If I recall correctly this is supposed to return short-living token for weights download. I am not sure what is the expiration period (checked and I think that it's 15s)